### PR TITLE
🐛 Fix board.py checks

### PR DESCRIPTION
**Pull request by Serg-Bartosh**

### CODE REVIEW

File: src/board.py [hunk 1]
```diff
@@ -39,4 +41,7 @@ def set_cells_with_value(self, points: list[Point], value: bool) -> None:
             if point.x >= self.height or point.y >= self.weight:
                 raise ValueError(f"{point} out of bounds!")
 
+            if point.x >= self.height or point.y >= self.weight:
```
Comment (alena-bartosh) [inline src/board.py]: Do you test it? Propose to to sleep and then think again

File: src/board.py [hunk 2]
```diff
@@ -20,6 +20,8 @@ def is_cell_empty(self, point: Point) -> bool:
         # if point out of bounds then return that cell is not empty
         if point.x >= self.height or point.y >= self.weight:
             return False
+        if point.x <= self.height or point.y <= self.weight:
```
Comment (alena-bartosh) [inline src/board.py]: Here as well

Comment (s373r) [inline src/board.py]: we need to check < 0 condition

File: src/board.py (hunk 1 above)
Comment (s373r) [inline src/board.py]: we need to check < 0 condition

**Review by s373r — CHANGES_REQUESTED** [review]

File: src/board.py (hunk 2 above)
Comment (s373r) [reply src/board.py]: + add a blank line before

**Review by alena-bartosh — DISMISSED** [review]

File: src/point.py [hunk 3]
```diff
@@ -5,3 +5,9 @@
 class Point:
     x: int
     y: int
+
+    def is_in_board(self, height: int, weight: int) -> bool:
+        return self.x >= height \
+               or self.y >= weight \
+               or self.x < 0 \
+               or self.y < 0
```
Comment (alena-bartosh) [inline src/point.py]: Noticed that it is not correct

Comment (s373r) [reply src/point.py]: > Noticed that it is not correct

yep

**Review by s373r — CHANGES_REQUESTED** [review]

**Review by alena-bartosh — APPROVED** [review]

**Review by alena-bartosh — DISMISSED** [review]

File: src/point.py [hunk 4]
```diff
@@ -5,3 +5,9 @@
 class Point:
     x: int
     y: int
+
+    def is_in_board(self, height: int, weight: int) -> bool:
+        return self.x <= height \
+               or self.y <= weight \
```
Comment (alena-bartosh) [inline src/point.py]:
```suggestion
        return self.x < height \
               or self.y < weight \
```

**Review by s373r — APPROVED** [review]

**Review by alena-bartosh — APPROVED** [review]

**Pull request merged**